HISTORY.rst

```diff
@@ -8,6 +8,7 @@ Unreleased Changes
   pygeoprocessing source distribution.  This fixes an issue where files
   matching a variety of extensions anywhere in the pygeoprocessing directory
   might be included with the source distribution.
+* Adding GDAL 3 support and dropping GDAL 2 support.
```

Comment:
Can you specify the things that will be different between GDAL 2 and 3? Seems like it's on the coordinate transformation order for unprojected SRSs. I remember in Slack you'd linked to a mailing list discussion or somesuch about why they decided to make this change. I think that'd be good to include here too.